lib/index.js

```diff
@@ -50,6 +50,11 @@ function compile(
     res = fmtCode
     resName = moduleName
   } catch (e) {
+    if (e[0][0] === "Diagnostic.Error") {
+      console.log('\x1b[31m%s\n\x1b[0m', '[PARSE ERROR]');
+      console.log(e[1])
+      process.exit(1)
+    }
```

Comment:
I had implemented to raise Error when parse-error detected, and then print an error to stdout.

But perhaps it might be better to reconsider signature of `Compile.compile` like `(string, string, bool) => (string, string)` to `(string, string, bool) => (string, string, list(string))`, where third element of tuple will represent `diagnostic errors`.

I'd say such function which has signature above can easy to test, but it is just an idea.

Reply:
Sounds like a great idea! Did you want to do it in this PR?

Reply:
I'm glad to hear it, and would like to do it other PR 🙏

Reply:
As I actually wrote it, I included it in this PR, because the difference of code
was smaller than I thought